src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -190,6 +190,13 @@ public function load(array $configs, ContainerBuilder $container)
         $container->getDefinition('security.access_listener')->setArgument(3, false);
         $container->getDefinition('security.authorization_checker')->setArgument(2, false);
         $container->getDefinition('security.authorization_checker')->setArgument(3, false);
+
+        if (!$config['password_policies']) {
+            $container->removeDefinition('security.password_policy_listener');
+        } else {
+            $policyServices = array_map(static fn (string $id) => new Definition($id), $config['password_policies']);
```

Comment:
I don't think this is correct: entries in the list of policies cannot directly map to a service definition. that'd mean not being allowed to configure their constructor arguments for example.
Instead, we could use `new Reference`, but I'm wondering if we're not missing a DI tag to do the link, and use `tagged_iterator()`

Reply:
Correct. I modified the extension and my service definitions in the test.
The use of a tag is a good idea, but can we detect if a service is tagged in this method?

Reply:
not in this method. We would have to do it in a compiler pass (or use `tagged_iterator` without checking things)